### ISSUE TITLE
Training files

### ISSUE THREAD

**Issue by GriffinLiang** [doubledaibo/drnet#2]
The prototxt files seem incomplete as the learning rates for many of the layers are zeros. It will be appreciated if more details for training stage are provided.

**Comment by doubledaibo** [comment]
Hi Griffin,
    The prototxt files are used to demonstrate the structures of networks, and should not be used directly for training. I've added the training procedure to README.md. And you may need to operate the files including adjusting learning rates and adding dropout layers to retrain the models.

**Comment by GriffinLiang** [comment]
Thanks. BTW, how to set the learning rate of each layer in the following step? {train A in isolation, train S in isolation, train A + S in isolation, train A + S + D jointly}

**Comment by yanxp** [comment]
@GriffinLiang  Have you started to train the model?

**Comment by GriffinLiang** [comment]
Nope. Only testing.

**Comment by yanxp** [comment]
Do you have the trained model? How to test?

**Comment by GriffinLiang** [comment]
@yanxp The download links are: https://github.com/doubledaibo/drnet/tree/master/snapshots

**Comment by yanxp** [comment]
@GriffinLiang  Ok,Thank you. However I meet a problem:File "tools/test_predicate_recognition.py", line 185, in <module>
    test_image_paths = json.load(open(args.image_paths))
What is the first parameter? How to set it? Thanks.

**Comment by GriffinLiang** [comment]
I am not sure whether the author has released it. So you'd better generate the file by yourself.
